docs/source/contrib/engines.rst

```diff
@@ -4,29 +4,26 @@ ignite.contrib.engines
 Contribution module of engines and helper tools:
 
 ignite.contrib.engines.tbptt
+-----------------------------
 
 .. currentmodule:: ignite.contrib.engines.tbptt
 
 .. autosummary::
     :nosignatures:
     :autolist:
 
+.. automodule:: ignite.contrib.engines.tbptt
+    :members:
+
 ignite.contrib.engines.common
+-----------------------------
 
-.. currentmodule:: ignite.contrib.engines.common
+.. seealso:: Common training setup functions are in :mod:`~ignite.contrib.engines.common`.
+
+.. currentmodule:: ignite.contrib.engines
 
 .. autosummary::
     :nosignatures:
-    :autolist:
-
-Truncated Backpropagation Throught Time
----------------------------------------
-
-.. automodule:: ignite.contrib.engines.tbptt
-   :members:
-
-Helper methods to setup trainer/evaluator
------------------------------------------
+    :toctree: ../generated
```

Comment:
@vfdev-5 should this page reverted to previous version?

Reply:
Yes, I liked the previous version.